exporter/otlp/lib/opentelemetry/exporter/otlp/exporter.rb

```diff
@@ -31,8 +31,19 @@ class Exporter # rubocop:disable Metrics/ClassLength
         WRITE_TIMEOUT_SUPPORTED = Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.6')
         private_constant(:KEEP_ALIVE_TIMEOUT, :RETRY_COUNT, :WRITE_TIMEOUT_SUPPORTED)
 
-        def initialize(endpoint: config_opt('OTEL_EXPORTER_OTLP_TRACES_ENDPOINT', 'OTEL_EXPORTER_OTLP_ENDPOINT', default: 'https://localhost:4317/v1/traces'), # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
+        def self.ssl_verify_mode
```

Comment:
💭 _We may have an opportunity here for a [refactoring](https://github.com/open-telemetry/opentelemetry-ruby/pull/743/files#diff-e00633625fe806f1deb53b278b99ddc078438e19f95f3f508e81d1a36f1024ecR18) but it is also small enough that it will likely cause more confusion than clarity._

Reply:
yeah, I think it's probably worth doing this for the next exporter that needs it. Maybe for the Zipkin exporter?

Reply:
I'm there's a few other utilities I've spotted that could be extracted into a base class or mixin for HTTP exporters. There's retry behaviour in the OTLP exporter that could be extract for instance.